db/migrate/20190130090437_add_crop_to_photographings.rb

```diff
@@ -7,4 +7,9 @@ def change
       p.set_crop && p.save!
     end
   end
+  class Photographing < ApplicationRecord
```

Comment:
While this works, I tend to avoid classes in schema migrations/management to avoid this; and push data migrations to rake tasks or rely on raw sql.